js/constructor.js

```diff
@@ -28,22 +32,39 @@ Store.prototype.cookieSoldHour = function(){
         }
     }  
 };
-Store.prototype.render = function(){
-    this.custHour();
-    this.cookieSoldHour();
-    var ulEl = document.getElementById(this.id);
-    for (var i=0; i<=this.storeHours+1; i++){
-        //create element
-        var liEl = document.createElement('li');
-        //give it content
-        liEl.textContent = hours[i] + ': '+ this.cookieSoldDay[i];
-        //append to DOM
-        //parent.appendChild(child)
-        ulEl.appendChild(liEl);
+Store.prototype.render = function() {
+    // create tr
+    var trEl = document.createElement('tr');
+    // create td
+    var tdEl = document.createElement('td');
+    // give td content (Name for an individual store)
+    tdEl.textContent = this.name;
+    // append the td
+    trEl.appendChild(tdEl);
+
+    for(var i=0; i<hours.length; i++){
+    // create td
+    tdEl = document.createElement('td');
+    // give td content (Name for an individual store)
+    tdEl.textContent = this.cookieSoldDay[i];
+    // append the td
+    trEl.appendChild(tdEl);
     }
-};
+    storeTable.appendChild(trEl);
+} 
 new Store('alkiStore', 'Alki Store', 2, 16, 4.6);
 new Store('pikeStore','Pike Store', 23, 65, 6.3,);
 new Store('seaTacStore', 'SeaTac Store', 3, 24, 1.2);
 new Store('capHillStore', 'Capital Hill Seattle', 23, 65, 6.3);
 new Store('seaCenterStore','Seattle Center Store', 11, 38, 3.7);
+
+function render2(){
```

Comment:
This function should not be named `render2` because that gives hardly any indication of what it does. Since it renders the table header, it should probably be named something to that effect.